README.md

```diff
@@ -44,6 +44,25 @@ This first builds the common and testrail-integration submodules and sets maven
 
 To run only specific runner (Java) class test use `-DfailIfNoTests=false` and `-Dtest=<TestName>`.
 
+## Configuring Tests Using System Properties
+
+You can configure various aspects of the tests using system properties. This is especially useful for providing credentials and environment-specific settings without modifying the code.
+
+```
+mvn test -DargLine="-Dkarate.env=snapshot -DbaseUrl=https://custom-okapi-url:9130 -Dadmin.tenant=mytenant -Dadmin.name=custom_admin -Dadmin.password=secret_password -DprototypeTenant=mytenant"
+```
+
+The following system properties are supported:
+- `baseUrl`: URL of the Okapi instance (e.g., http://localhost:9130)
```

Comment:
Since Okapi is not being used it makes sense to remove or replace it with platform.